src/message/displaywindowComponent.js

```diff
@@ -262,6 +299,8 @@ ngeo.message.displaywindowComponent.component('ngeoDisplaywindow', {
   bindings: {
     'clearOnClose': '<?',
     'content': '=?',
+    'contentTemplate': '<',
+    'contentScope': '<',
```

Comment:
You must add `?` symbol to the bindings otherwise it's a mandatory bindings and you will get errors with example's window 1 and 2.